src/SWP/Bundle/WebRendererBundle/Exception/NoThemeException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Web Renderer Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\WebRendererBundle\Exception;
+
+class NoThemeException extends \Exception
```

Comment:
Agree. In case you silence the exception and pass it to the listener it not required